setup.sh

```diff
@@ -1,29 +1,11 @@
 #!/bin/bash
-
-# e.g. this:
-#export PATH=/home/cwret/procmail-3.22/new:${PATH}
-#source /home/cwret/CMT/setup.sh
-#source /home/cwret/root/bin/thisroot.sh
-#module load 
-#export CUDAPATH=${CUDA_HOME}
-#export MACH3_DATA=/home/cwret/P6Data
-#export MACH3_MC=/home/cwret/P6MC
-
 #################################################################
-
-if [[ $HOSTNAME == *gpu.rl.ac.uk ]]; then
-  # Add lockfile to path (required on emerald!) (commented out for those not on EMERALD)
-  export PATH=$PATH:/home/oxford/eisox159/procmail/bin
-
-  # setup root on EMERALD (commented out for those not on EMERALD)
-  source /home/stfc/eisext13/root/bin/thisroot.sh
-  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/stfc/eisext13/root/lib
-  export PATH=$PATH:/home/stfc/eisext13/root/bin
-  export ROOTLIBDIR=/home/stfc/eisext13/root/lib
-  export ROOTINCLUDEDIR=/home/stfc/eisext13/root/include
-  module load cuda/8.0.44
-  export CUDAPATH=${CUDA_HOME}
-fi
+source ./build/bin/setup.MaCh3DUNE.sh
+source ../MaCh3_core_dev/build/bin/setup.MaCh3.sh
```

Comment:
Is any of this actually needed? Seems hardcoded to someones (presumably Ed's?) setup with hardcoded relative paths

Reply:
Removed